main_script.py

```diff
@@ -0,0 +1,125 @@
+from internal_functions.dna_rna_tools import transcribe
+from internal_functions.dna_rna_tools import reverse
+from internal_functions.dna_rna_tools import complement
+from internal_functions.dna_rna_tools import reverse_complement
+from internal_functions.protein_analysis_tool import molecular_weight
+from internal_functions.protein_analysis_tool import one_letter_to_three
+from internal_functions.protein_analysis_tool import get_amino_acid_sum
+from internal_functions.protein_analysis_tool import codon_optimization
+from internal_functions.protein_analysis_tool import length
+from internal_functions.protein_analysis_tool import name_transform
+from internal_functions.protein_analysis_tool import is_amino_acid
+from internal_functions.protein_analysis_tool import brutto_count
+from internal_functions.protein_analysis_tool import is_length_divisible_by_3
+from internal_functions.protein_analysis_tool import is_amino_acid_three_letter
+from internal_functions.work_with_fastq import gc_count_filter
+from internal_functions.work_with_fastq import length_filter
+from internal_functions.work_with_fastq import quality_threshold_filter
+
+
+def run_dna_rna_tools(*args: str) -> str:
+    """
+    run_dna_rna_tools is a utility for working with sequences of nucleic acids
+
+    Arguments:
+    - args (str): an arbitrary number of arguments with DNA or RNA sequences,
+                      as well as the name of the procedure to be performed:
+                      transcribe, reverse, complement, reverse_complement
+    Return:
+    - the line with the result of the operation
+    """
+    sequences, operation = args[0:-1], args[-1].lower()
+    type_of_operation = {'transcribe': transcribe(sequences),
+                         'reverse': reverse(sequences),
+                         'complement': complement(sequences),
+                         'reverse_complement': reverse_complement(sequences)}
+    for sequence in sequences:
+        for nucleotide in sequence:
+            if 't' == nucleotide.lower() or 'u' == nucleotide.lower():
+                raise ValueError('A nonexistent sequence has been introduced')
+            else:
+                return type_of_operation.get(operation)
```

Comment:
метод get можно использовать, но обычно, чтобы получить значение по ключу, используют такую запись:
```suggestion
                return type_of_operation[operation]
```